og.install

```diff
@@ -74,3 +74,10 @@ function og_schema() {
 
   return $schema;
 }
+
+/**
+ * Implements hook_install().
+ */
+function og_install() {
+  \Drupal\og\Og::groupResolverHandler()->updateConfigStorage();
+}
```

Comment:
This shouldn't be needed. If I look at other modules that provide configurable plugins (e.g. block and views) then they also don't initialize their plugins on `hook_install()`. This can probably be replaced by default config, or simply by initializing on first use.